node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
Oh I want to say the NAME of package is non-ideal. We are basically having `node_agent/node_agent` but call it `node_agent/na` to avoid redundancy.